doc/UsersGuide/source/CAPE2020.rst

```diff
@@ -0,0 +1,166 @@
+
+The original July 2020 CAPE case illustrated a shortcoming of the Global Forecast System (GFS) v16 --- low Convective Available Potential Energy (CAPE) predictions during summertime (:cite:t:`SunEtAl2024`). :cite:t:`SunEtAl2024` (2024) used this case study to investigate the low CAPE bias in the GFS and determined that "the GFS simulates smaller surface latent heat flux and larger surface sensible heat flux than the observations" due to "slightly drier-than-observed soil moisture" within the offline Global Data Assimilation System (GDAS) initial conditions used in the study. This resulted in less latent heat and moisture being fed back to the lower levels of the atmosphere and ultimately changed the overall vertical profile of the atmosphere, which lowered CAPE values relative to the older GFS v15.2. 
+ 
+The UFS WM and its subcomponents have undergone signficant changes since the original July 2020 CAPE case study was posted and since :cite:t:`SunEtAl2024`'s experiment, so the current GFS v16 CAPE bias may have shifted. However, users may still wish to run this case and then experiment with different (potentially user-generated) initial conditions, a coupled land surface model (LSM), or other factors to explore factors that improve or worsen CAPE bias. Additionally, :cite:t:`SunEtAl2024`'s findings only apply to this case study, so users may wish to expand their research to include other warm-season cases. 
+
+============================================
+Obtaining Data for the July 2020 CAPE Case
+============================================
+
+.. include:: ./doc-snippets/hsd_data.rst
+
+.. _chgres-data:
+
+User-Generated Data
+---------------------
+
+.. attention::
+
+   The following instructions only apply to users with access to :term:`HPSS` on :term:`RDHPCS`. In the future, there are plans to expand options for access to raw ICs data for other users. 
+
+Users who have access to :term:`HPSS` can generate initial conditions (:term:`ICs`) for a particular forecast case and resolution by downloading the raw GFS data and converting it to the appropriate version using the the UFS_UTILS ``gdas_init`` utility. ``gdas_init`` pulls the input data required by ``chgres_cube`` from HPSS and then runs the ``chgres_cube`` utility to create coldstart initial conditions for the desired resolution and number of vertical levels. Users who already have access to raw GFS ICs can use just the ``chgres_cube`` utility to perform the conversion on their existing data. Users may wish to refer to the `UFS_UTILS User's Guide <https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html>`_ for more information. 
+
+.. note:: 
+   
+   In order to generate all necessary configuration, data, input, and fix files to run in this method, the user first needs to run the base RT script using ``./rt.sh -n`` to set up the run directory. 
```

Comment:
@gspetro-NOAA i would remove this and simply say that the user should copy the gfs*.nc and sfc*.nc files generated by gdas_init into the `INPUT` directory of their  test case's run directory (e.g., for 2020_CAPE w/ Intel, that directory would be in ufs-weather-model/tests-dev/run_dir/2020_CAPE_intel). the ufs-weather-model/tests-dev/run_dir path is a symlink to the actual run directory location, so the user should cd into the absolute path and copy the gfs*.nc and sfc*.nc files into the `INPUT` directory therein.

Reply:
But this implies that they have already run the case w/default settings, right...?

Reply:
yes, they can follow this "copy" method when they already have run a case and just want to try new ICs and a new start time. otherwise, if they want to configure there own case with new ICs from scratch, they will need to make sure their ICs are copied in via the relevant `*.IN` file from tests-dev/test_cases/exp_conf.